apps/website/components/MicaComplianceTable.tsx

```diff
@@ -0,0 +1,259 @@
+import { Panel, Text, Tooltip } from '@siafoundation/design-system'
+import { MicaIndicatorObject } from '../content/mica'
+import {
+  CloudLightning32,
+  CloudUpload32,
+  Earth32,
+  Information16,
+  TrashCan32,
+} from '@carbon/icons-react'
+
+type MicaComplianceTableCellGroupLayoutProps = {
+  title: string
+  description: string
+  displayValue: string
+  last?: boolean
+}
+
+function MicaComplianceTableCellGroupLayout({
+  title,
+  description,
+  displayValue,
+  last,
+}: MicaComplianceTableCellGroupLayoutProps) {
+  return (
+    <>
+      <td
+        className={`text-left py-4 ${
+          !last && 'border-b-2 dark:border-b-graydark-100'
```

Comment:
Add both light and dark mode classes: `border-gray-100 dark:border-graydark-100` here and for the other instances.